pom.xml

```diff
@@ -2021,7 +2056,7 @@
         <!--<identity.carbon.auth.signedjwt.version>5.1.1</identity.carbon.auth.signedjwt.version>-->
         <!--<identity.userstore.cassandra.version>5.1.1</identity.userstore.cassandra.version>-->
         <!--<identity.agent-entitlement-filter.version>5.1.1</identity.agent-entitlement-filter.version>-->
-
+        <org.wso2.identity.remotefetch.version>0.7.0-SNAPSHOT</org.wso2.identity.remotefetch.version>
```

Comment:
Why are we referring to **SNAPSHOT** version here?

Reply:
Fixed in later PR, 
https://github.com/wso2/product-is/pull/8077/files#diff-600376dffeb79835ede4a0b285078036R2054